assets/css/footer.css

```
@@ -0,0 +1,375 @@
+.footer{
+    display: flex;
+    width: 100vw;
+    min-height: 950px;
+    flex-direction: column;
+}
+button{
+    cursor: pointer;
+}
+.widgets{
+    width:100%;
+    min-height: 850px;
+    background-color: #212331;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+   
+}
+.widgets-wrap{
+    max-width: 1200px;
+    min-height: 650px;
+}
+.menuSocials{
+    width: 100%;
+    min-height: 100px;
+    background-color: #181828;
+}
+.widgets-wrap-item{
+    min-height: 650px;
+    padding: 50px 0;
+}
+.widgets-wrap-col--ct ,.widgets-wrap-col--contact{
+    width: 100%;
+   
+}
+.widgets-wrap-col--ct li{
+    padding-top: 20px;
+}
+.widgets-wrap-col{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    min-height: 650px;
+    color: #99a9b5;
+    align-items: inherit;
+    margin-bottom: 50px;
+}
+.trig-box{
+    display: flex;
+}
+.trig-bottomleft {
+    box-sizing: border-box;
```

box-sizing: border-box không set riêng lẻ từng phần tử mà phải set cho toàn bộ các phần tử của trang. em bỏ các dòng này và viết lại như này để tất cả các phần tử html đều có box-sizing: border-box em nhé
```suggestion
    * {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}
```